### ISSUE TITLE
alternate distribution of blockchain state

### ISSUE THREAD

**Issue by faddat** [virtualeconomy/v-systems#246]
@speedmax and I were talking and what are the ideas that we had upon which I think would be very beneficial to the project overall is the creation of pre-synced Docker images. 

The goal here is just to reduce the amount of time that it takes for a developer to actually begin work.

My planned technique for creating these is:

* Run a node with systemd
* get it synced
* stop the node once a week
* Create a [docker image](https://hub.docker.com/repository/docker/faddat/vsys) that uses COPY to move the chain state into the image

I'd like to do this for both testnet and mainnet and am mainly looking for feedback on implementation.

**Comment by speedmax** [comment]
Monthly archive of chain data folder would be sufficient.

**Comment by faddat** [comment]
Yes that would be sufficient, however it's quite simple to automate things with Docker and so I figured why not do it daily and push that to Docker hub daily.
